src/lib/replication/log_store/repl_log_store.cpp

```diff
@@ -73,6 +81,17 @@ void ReplLogStore::end_of_append_batch(ulong start_lsn, ulong count) {
         for (auto const& rreq : *reqs) {
             if (rreq) { rreq->add_state(repl_req_state_t::LOG_FLUSHED); }
         }
+
+        for (auto const& rreq : *reqs) {
+            if ((rreq == nullptr) || (!rreq->has_linked_data())) { continue; }
+            LOGINFO("Raft Channel: Data after future wait: rreq=[{}]", rreq->to_compact_string());
+        }
+    } else if (!proposer_reqs->empty()) {
+        LOGINFO("Raft Channel: end_of_append_batch, I am proposer, only flush log s from {} , count {}", start_lsn, count);
+        // Mark all the reqs also completely written
+        for (auto const& rreq : *reqs) {
```

Comment:
Don't this need to be `*proposer_reqs`?

Reply:
Also we need to call `HomeRaftLogStore::end_of_append_batch(start_lsn, count);` right?